.github/workflows/coverage.yml

```diff
@@ -37,7 +37,7 @@ jobs:
 
       - name: Test NetworkX
         run: |
-          pytest --cov=networkx --runslow -n auto --doctest-modules --durations=20 --pyargs networkx
+          pytest --cov=networkx --runslow -n auto --doctest-modules --durations=20 --pyargs --mpl networkx
```

Comment:
Let's leave the image tests out of the coverage job, which is already our longest-running CI job and there should be no additional coverage boosts from the image comparison tests.